### PR TITLE
Do not show "no geography" in the meta detail list of geographies

### PR DESCRIPTION
**Pull request by PatFawbertMills**
# What's changed
- Only display "International" in the list of associated geographies
- Strictly not to display "No Geography" at any point

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: src/constants/systemGeos.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
+export const SYSTEM_INTERNATIONAL_CODE = "xab";
+export const SYSTEM_INTERNATIONAL_NAME = "international";
+
 export const systemGeoCodes = [
   "xaa", // No Geography
-  "xab", // International
+  SYSTEM_INTERNATIONAL_CODE,
```
Comment (katybaulch) [inline src/constants/systemGeos.ts:6]: Are we happy with this change affecting CPR/CCLW as well?

**Review by odrakes-cpr — APPROVED** [review]

File: src/constants/systemGeos.ts [hunk 2]
```diff
@@ -1,6 +1,9 @@
+export const SYSTEM_INTERNATIONAL_CODE = "xab";
+export const SYSTEM_INTERNATIONAL_NAME = "international";
+
 export const systemGeoCodes = [
   "xaa", // No Geography
```
Comment (katybaulch) [inline src/constants/systemGeos.ts:5]: Why isn't this a const too, if we're pulling XAB out into a const?

Comment (PatFawbertMills) [reply src/constants/systemGeos.ts]: Should be, I was being lazy

File: src/constants/systemGeos.ts (hunk 1 above)
Comment (PatFawbertMills) [reply src/constants/systemGeos.ts]: That was the requirement as I understood it. Previously we were not displaying either "International" or "No Geography"

Comment (katybaulch) [reply src/constants/systemGeos.ts]: Awesome - just wanted to check!

**Review by katybaulch — APPROVED** [review]
lgtm

**Pull request merged**